javatests/arcs/sdk/spec/EntitySpecTest.kt

```diff
@@ -325,10 +330,11 @@ class EntitySpecTest {
      * Stores the given [Bar] entity in a collection, and then creates and returns a reference to
      * it.
      */
-    private suspend fun createBarReference(bar: Bar): Reference<Bar> {
-        harness.bars.store(bar)
-        return harness.bars.createReference(bar)
-    }
+    private suspend fun createBarReference(bar: Bar): Reference<Bar> =
+        withContext(harness.bars.dispatcher) {
+            harness.bars.store(bar)
+            return@withContext harness.bars.createReference(bar)
```

Comment:
Do you need the `return@withContext` here? Does it work if you just leave `createReference` as the last expression of the lambda?

Reply:
It does work.. I'm torn on which version is more readable. I'll drop it though :)